tests/anmykh/test_collections.py

```diff
@@ -0,0 +1,22 @@
+some_str = "abcd"
+some_tuple = "a", "b", "c", "d"
+some_list = ["a", "b", "c", "d"]
+some_set = {"a", "b", "c", "d"}
+some_dict = {"a": 123, "b": 456, "c": 789, "d": 100}
+b = [1, 2, 3, 4]
+
+
+def test_modify_collection():
+    b[3] = "abc"
+    assert b[3] == "abc"
```

Comment:
do not change in tests global objects, since you spoil isolation between tests

lets say test a and test b both use  some_list = [1,2, 3]

test a removes item from list
test b expects to see 3 items in list and fails

Reply:
create objects at once in test